### PR TITLE
feat(extra-natives/five): implement [GET/SET]_GLOBAL_PASSENGER_MASS_MULTIPLIER

### PR DESCRIPTION
**Pull request by Explooosion-code**
### Goal of this PR
Adding a getter and setter to control the mass of passengers. I.E if you have a racing server you might want to have passengers not decrease performance of vehicle and you might want to 
```lua
 SetGlobalPassengerMassMultiplier(0.0)
```

...

### How is this PR achieving the goal
By setting/getting the value that GTA is using for decreasing performance of vehicle depending on passenger count

...

### This PR applies to the following area(s)
Fivem, Extra Natives

...

### Successfully tested on

**Game builds:** 1604, 2060, 2189, 3095, 3258

**Platforms:** Windows

### Checklist

- [x] Code compiles and has been tested successfully.
- [x] Code explains itself well and/or is documented.
- [x] My commit message explains what the changes do and what they are for.
- [x] No extra compilation warnings are added by these changes.

### CODE REVIEW

**Comment by qalle-git** [comment]
yes, please

**Comment by prikolium-cfx** [comment]
Hi. Can you please rename natives to SET/GET_GLOBAL_PASSENGER_MASS_MULTIPLIER and also you need to reset it on disconnect from the server in OnKillNetwork to its default value. You can check how our custom natives work now.

File: code/components/extra-natives-five/src/VehicleExtraNatives.cpp [hunk 1]
```diff
@@ -1518,6 +1525,25 @@ static HookFunction initFunction([]()
...
+		{
+			weight = 0.0;
+		}
+		trace("Setting weight %.6f was %.6f\n", weight, *PassengerMassPtr);
+		*PassengerMassPtr = weight;
+	});
+
+	fx::ScriptEngine::RegisterNativeHandler("GET_GLOBAL_PASSENGER_MASS_MULTIPLIER", [](fx::ScriptContext& context)
+	{
+		trace("Weight is %.6f\n", *PassengerMassPtr);
```
[the remark this answers is not in the document]
Comment (Explooosion-code) [reply code/components/extra-natives-five/src/VehicleExtraNatives.cpp:1542]: Yeah, my bad, forgot to remove them after testing

File: ext/native-decls/SetGlobalPassengerMassMultiplier.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+## SET_GLOBAL_PASSENGER_MASS_MULTIPLIER
+
+```c
+void SET_GLOBAL_PASSENGER_MASS_MULTIPLIER(float massMul);
+```
+
+## Parameters
+* **massMul**: Weight of each passenger (not counting the driver) as a percentage of vehicle mass. Is applied to all vehicles. Default value is 0.05
+
+## Return value
```
Comment (PsychoShock) [inline ext/native-decls/SetGlobalPassengerMassMultiplier.md:15]: No need to have this if no return value.

**Comment by prikolium-cfx** [comment]
Rebased and merged.

**Pull request closed without merging**